Tools-Override/tests.targets

```diff
@@ -28,13 +28,15 @@
 
   <PropertyGroup>
     <XunitRuntimeConfig>$(ToolsDir)\xunit.console.netcore.runtimeconfig.json</XunitRuntimeConfig>
-    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT'">%RUNTIME_PATH%\</TestRuntimeEnvVar>
+    <TestRuntimeEnvVar Condition="'$(OS)' == 'Windows_NT' and '$(RUNTIME_PATH)' != ''">%RUNTIME_PATH%\</TestRuntimeEnvVar>
```

Comment:
Is it really necessary to be using environment variables outside of the build like this? It seems fragile to be using non-expanded env strings like this- particularly at the beginning of a chain of property building.

I can condition on running in VS I suppose- that's probably the least likely to break.